R/create_protocol.R

```diff
@@ -71,8 +71,7 @@
 #' the target folder beneath `src`.
 #' Besides Rmarkdown files, this target folder will also contain files needed to render
 #'  to a Bookdown gitbook such as a `_bookdown.yml`.
-#' Additionally, a `NEWS.Rmd` file will be written to the target folder which
```

Comment:
I'd keep the NEWS.Rmd filename. And set the files to render (and their order) via `_bookdown.yml`. See https://bookdown.org/yihui/bookdown/usage.html